### ISSUE TITLE
antenna positions metadata

### ISSUE THREAD

**Issue by rlbyrne** [EoRImaging/FHD#131]
FHD metadata does not pull antenna positions from the uvfits file. This is mostly an issue for simulation. Simulations that use the MWA beam report MWA antenna positions even if the data has different antenna positions.

**Comment by rlbyrne** [comment]
@PyxieLouStar is this resolved by #285?

**Comment by PyxieLouStar** [comment]
I don't think so; the way antenna numbers were pulled from the uvfits was changed but nothing with antenna positions should have been impacted.
